tests/platform_tests/thermal_control_test_helper.py

```diff
@@ -229,9 +230,10 @@ def check_cli_output_with_mocker(dut, mocker_object, command, max_wait_time, key
     """
     time.sleep(max_wait_time)
 
-    result = dut.show_and_parse(command)
-    assert len(result) > 0, "Run and parse output of command '{}' failed".format(command)
-    return mocker_object.check_result(result)
+    parsed_output = dut.show_and_parse(command)
+    assert len(parsed_output) > 0, "Run and parse output of command '{}' failed".format(command)
+    result = mocker_object.check_result(parsed_output)
+    pytest_assert(result, 'FAN mock data mismatch')
```

Comment:
It might be fan mock data or thermal mock data. So for mocking thermal data, the assert message here might be confused.

Reply:
@Junchao-Mellanox  I fixed the assertion error message, see it in my next commit